cmd/trace-agent/config/config.go

```diff
@@ -597,3 +594,44 @@ func acquireHostnameFallback(c *config.AgentConfig) error {
 	log.Debugf("Acquired hostname from core agent (%s): %q.", c.DDAgentBin, c.Hostname)
 	return nil
 }
+
+// SetHandler returns handler for runtime configuration changes.
+func SetHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPost {
+			http.NotFoundHandler()
+			return
+		}
+		for set, values := range req.URL.Query() {
+			value := ""
+			for i := range values {
```

Comment:
I think that having multiple values is likely an error. But that's fine, we should do a best effort to support a reasonable outcome. I think that if multiple values are provided, we should just take the last one. So, I'd propose something like this:
```suggestion
			if len(values) == 0 {
			    continue
			}
			value := values[len(values-1)] // account for multiple settings and take the last one
```
So we always take the last value (probably 99.9% of the time it will only be one). Imagine this scenario:
```
http://localhost:8126/config/set?log_level=info&log_level=debug
```
It makes sense to assume that the last setting is the desired one.

But again, this is an exaggerated example because having more than one doesn't make sense most of the time.

Reply:
and if the last one is an empty string?

Reply:
Good point. I think it's fine if nothing happens in that case. It would be odd to just take the last non-empty one. Perhaps we should log something at debug level.

Reply:
I guess, let's make it very simple. Considering we only have one field to change, it might be modified in the future. I'll take the last value for now